spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestMerge.java

```diff
@@ -1609,8 +1611,14 @@ public synchronized void testMergeWithSnapshotIsolation()
     createOrReplaceView("source", Collections.singletonList(1), Encoders.INT());
 
     sql(
-        "ALTER TABLE %s SET TBLPROPERTIES('%s' '%s')",
-        tableName, MERGE_ISOLATION_LEVEL, "snapshot");
+        "ALTER TABLE %s SET TBLPROPERTIES('%s', '%s', '%s', '%s', '%s', '%s')",
```

Comment:
I think you've got too many commas here. I believe this should be `('%s' '%s', '%s' '%s', '%s' '%s')` and same for the other Spark test